input/templates/my-account-address-book.hbs

```diff
@@ -34,61 +34,34 @@
                       <div class="box-addresses-title">
                         {{i18n "my-account:addressBook.defaultAddresses.title"}}
                       </div>
+                      {{#if content.defaultShippingAddress}}
                       <div class="address-book-dark-box">
-                        <div class="address-book-dark-box-title">{{i18n "my-account:addressBook.normalAddress.title"}}</div>
-                        <div>{{content.shippingAddress.name}}</div>
-                        <div>{{content.shippingAddress.address}}</div>
-                        <div>{{content.shippingAddress.region}}</div>
-                        <div>{{content.shippingAddress.postalCode}}</div>
-                        <div>{{content.shippingAddress.country}}</div>
-                        <div class="address-box-edit-btn">
-                          <a href="{{@root.meta._links.editMyAddressBook.href}}"><img src="{{meta.assetsPath}}/img/edit-1.png" alt="edit icon">{{i18n "my-account:buttons.edit"}}</a>
-                        </div>
+                        <div class="address-book-dark-box-title">{{i18n "my-account:addressBook.defaultShippingAddress.title"}}</div>
+                        {{> my-account/my-address-book-address address=content.defaultShippingAddress}}
                       </div>
+                      {{/if}}
+                      {{#if content.defaultBillingAddress}}
                       <div class="address-book-dark-box">
                         <div class="address-book-dark-box-title">{{i18n "my-account:addressBook.billingAddress.title"}}</div>
-                        <div>{{content.billingAddress.name}}</div>
-                        <div>{{content.billingAddress.address}}</div>
-                        <div>{{content.billingAddress.region}}</div>
-                        <div>{{content.billingAddress.postalCode}}</div>
-                        <div>{{content.billingAddress.country}}</div>
-                        <div class="address-box-edit-btn">
-                          <a href="{{@root.meta._links.editMyAddressBook.href}}"><img src="{{meta.assetsPath}}/img/edit-1.png" alt="edit icon">{{i18n "my-account:buttons.edit"}}</a>
-                        </div>
+                        {{> my-account/my-address-book-address address=content.defaultBillingAddress}}
                       </div>
+                      {{/if}}
                     </div>
                     <div class="col-sm-6">
                       <div class="box-addresses-title">
                         {{i18n "my-account:addressBook.additionalAddress.title"}}
                       </div>
+                      {{#each content.addresses}}
```

Comment:
I'm not sure this was clear, but the other addresses are just a unlimited list of addresses. With the previous template you could only put 2. With this loop, we make sure all the addresses are shown.